www/source/docs/internals-bootstrapping.html.md

```diff
@@ -113,7 +113,7 @@ In this stage, we rebuild all the base packages needed by Habitat using the tool
 export STUDIO_TYPE=default
 export HAB_STUDIO_ROOT=/hab/studios/stage2
 export HAB_ORIGIN=core
-export HAB_DEPOT_URL=http://127.0.0.1:9632/v1
+export HAB_BLDR_URL=http://127.0.0.1:9632/v1
```

Comment:
Drop the `/v1`?

Reply:
Holy crap, good catch. These are ancient docs, that port doesn't even get used anymore :). I'll get this in the next one